### PR TITLE
Update titles to be consistent for new dbs as preview

### PR DESCRIPTION
**Pull request by stevenGravy**
Inconsistent titling and menu for indicating preview

### CODE REVIEW

File: docs/pages/database-access/guides/redis-aws.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Database Access with AWS ElastiCache and AWS MemoryDB for Redis
+title: Database Access with AWS ElastiCache and AWS MemoryDB for Redis (Preview)
```
Comment (zmb3) [inline docs/pages/database-access/guides/redis-aws.mdx:2]: Are ElastiCache and MemoryDB both in preview or is it just MemoryDB?

**Comment by ptgott** [comment]
@stevenGravy Would you like to get this one merged still?

**Comment by stevenGravy** [comment]
> @stevenGravy Checking on whether we want to go ahead with this one, since the last review was in July. Thanks!

These are still in preview so should be marked as such.

**Review by ptgott — APPROVED** [review]

File: docs/pages/database-access/guides/snowflake.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Database Access with Snowflake
+title: Database Access with Snowflake (Preview)
```
Comment (alexfornuto) [inline docs/pages/database-access/guides/snowflake.mdx:2]: If we adjusted the title in `config.json` for the other page, shouldn't we do the same for this one?

Comment (stevenGravy) [reply docs/pages/database-access/guides/snowflake.mdx:2]: @alexfornuto it does in `config.json`, just not in the page

Comment (alexfornuto) [reply docs/pages/database-access/guides/snowflake.mdx:2]: So it does... but the title there doesn't mention Redis. Not sure if it should or not, just saying.

Feel free to resolve this thread if the sidebar title shouldn't include Redis.

**Pull request closed without merging**